### PR TITLE
feat: options can be rendered as already selected OR-184

### PR DESCRIPTION
**Pull request by akomiqaia**

### CODE REVIEW

File: components/o-multi-select/src/tsx/multi-select.tsx [hunk 1]
```diff
@@ -1,16 +1,21 @@
...
+	label: string;
+	selected: boolean;
+}
+
 export function MultiSelect({
 	multiSelectOptions,
 	id,
 }: {
-	multiSelectOptions: string[];
+	multiSelectOptions: MultiSelectType[];
```
Comment (notlee) [inline components/o-multi-select/src/tsx/multi-select.tsx:10]: This is a breaking change. We need to release a new major or support both `string[]` and `MultiSelectType[]`

File: components/o-multi-select/src/js/multi-select-options.js [hunk 2]
```diff
@@ -92,14 +92,15 @@ function createOptionButton(option, index) {
...
  * @returns {HTMLElement} The newly created option element.
  */
-export function createOption(idBase, option, index) {
+export function createOption(idBase, option, index, selected = false) {
 	const optionEl = document.createElement('div');
 	optionEl.setAttribute('role', 'option');
 	optionEl.id = `${idBase}-${index}`;
 	optionEl.className = 'o-multi-select-option';
-	optionEl.setAttribute('aria-selected', 'false');
+	optionEl.setAttribute('aria-selected', selected);
```
Comment (notlee) [inline components/o-multi-select/src/js/multi-select-options.js:103]: I think this should be true/false?
https://developer.mozilla.org/en-US/docs/Web/API/Element/ariaSelected#value

**Review by notlee — CHANGES_REQUESTED** [review]
Nice work fixing this quickly for Marlon :) It would be good to add tests for this also

**Review by notlee — CHANGES_REQUESTED** [review]
You also need to write a Migration Guide and update the table in the README to release a major. Both work, but probably it would be easier to support both TSX interfaces 😀

**Comment by frshwtr** [comment]
Note from pairing: The enhanced experience doesn't update the core values when selected. Ideally it should do this if a form gets submitted off the back of this.

**Review by notlee — APPROVED** [review]

**Pull request merged**